addons/medical_treatment/XEH_preStart.sqf

```diff
@@ -1,3 +1,12 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+// Cache configured item conversions
+private _itemHash = [] call CBA_fnc_hashCreate;
+
+{
+    [_itemHash, configName _x, getArray _x] call CBA_fnc_hashSet;
+} forEach configProperties [configFile >> QGVAR(replaceItems)];
+
+uiNamespace setVariable [QGVAR(replaceItems), _itemHash];
```

Comment:
Might throw the security argument here.
Actually yes, I think that's relevant. Manipulating that array you can also inject things like "replace bandage with a M134 minigun"
You can replace any item in your inventory with anything. Def needs compileFinal.